CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# Contributing Guidelines
+
+This documentation contains a set of guidelines to help you during the contributing process. We are happy to welcome all the contributors who's willing to enhance/add features to this project. Feel free to contribute and remember, no contribution is too small.
+
+## Submitting Contributions :woman_technologist: :man_technologist:
+
+### Step 1 : Find an issue
+
+- Take a look at the Existing Issues or [create your own Issues!](#creating-an-issue)
+- Wait for the Issue to be assigned to you after which you can start working on it.
```

Comment:
`Kindly wait for the issue to be assigned to you. Once assigned you can get started with working on the same`